server/sonar-server/src/main/java/org/sonar/server/computation/issue/IssueComputation.java

```diff
@@ -47,13 +49,13 @@ public IssueComputation(RuleCache ruleCache, SourceLinesCache linesCache, ScmAcc
     this.diskIssuesAppender = issueCache.newAppender();
   }
 
-  public void processComponentIssues(ComputationContext context, String componentUuid, Iterable<BatchReport.Issue> issues) {
+  public void processComponentIssues(ComputationContext context, String componentUuid, Iterable<BatchReport.Issue> issues, @Nullable String defaultAssignee) {
```

Comment:
Adding such nullable parameters should be avoided. With such pattern in a few weeks this method will have more and more nullable params.